src/operator/src/statement/ddl.rs

```diff
@@ -121,6 +121,9 @@ impl StatementExecutor {
         // Because it will be used directly while compaction.
         if let Some(schema_options) = schema_options {
             for (key, value) in schema_options.extra_options.iter() {
+                if key.starts_with("compaction.") {
```

Comment:
I think we must update the comment above too.

Reply:
done, thx